webpages/settings/index.html

```diff
@@ -378,7 +378,7 @@ <h1 v-cloak>{{ msg("settings") }}</h1>
         </p>
         <p>
           <a
-            href="./licenses.html?libraries=icu-message-formatter,vue,color-picker-web-component,comlink,Sora,fuse,idb,sortable,Roboto"
+            href="./licenses.html?libraries=icu-message-formatter,vue,color-picker-web-component,comlink,Sora,fuse,idb,sortable,Roboto,scratch-gui"
```

Comment:
All the other licenses listed here are for the settings page. Is there a specific reason for including `scratch-gui`?

Reply:
Good catch

Reply:
The settings page uses the `download-blob.js` module from scratch-gui.

Reply:
@mxmou Can we remove that dependency instead? It's unnecessary :P 
The if statements in that file will never be truthy in environments where SA can run